srcpkgs/ytcc/template

```diff
@@ -1,19 +1,31 @@
 # Template file for 'ytcc'
 pkgname=ytcc
-version=1.8.2
-revision=4
+version=2.0.1
+revision=1
 build_style=python3-module
+make_use_env=yes
 hostmakedepends="gettext python3-setuptools"
-depends="python3-SQLAlchemy python3-feedparser python3-lxml python3-youtube-dl
- mpv"
+depends="mpv python3-SQLAlchemy python3-click python3-feedparser python3-lxml
+ python3-youtube-dl"
+checkdepends="python3-pytest python3-click youtube-dl ffmpeg mpv"
 short_desc="Cmdline tool to track your youtube channels"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/woefe/ytcc"
 distfiles="https://github.com/woefe/ytcc/archive/v${version}.tar.gz"
-checksum=360db561bb0278b7f326fe3da9b012e738e1eceec9031c66b3749207f8091283
+checksum=edb5367209a8c465c6b6f4f01705430602f25ca29b318dc18e42c139d2fe1efc
+
+pre_check() {
+	# Needs XDG_RUNTIME_DIR variable
+	vsed -e '/test_play_video/,/assert len/d' -i test/test_cli.py
```

Comment:
Why not `export XDG_RUNTIME_DIR=$(mktemp -d)` then `rm -rf $XDG_RUNTIME_DIR` ?

Reply:
DOesn't give that error anymore but still get:
WARNING  ytcc.cli:cli.py:408 The video player terminated with an error. The last video is not marked as watched

Which I sorta believe is maybe to be expected since inside chroot?